build-system/tasks/e2e/selenium-webdriver-controller.js

```diff
@@ -134,6 +137,16 @@ class SeleniumWebDriverController {
     return webElements.map(webElement => new ElementHandle(webElement, this));
   }
 
+  /**
+   * @return {!Promise<!ElementHandle<!WebElement>>}
+   * @override
+   */
+  async getActiveElement() {
+    const activeElement = await this.driver.executeScript(
```

Comment:
Can you use `this.driver.switchTo.activeElement()` instead? 
https://seleniumhq.github.io/selenium/docs/api/javascript/module/selenium-webdriver/lib/webdriver_exports_TargetLocator.html#activeElement

Reply:
That's a good idea. I was originally under the impression that the `switchTo().activeElement()` would cause future calls e.g. `findElement` to be referenced from under that node's tree. But the API docs don't seem to imply that